modules/apps/callbacks/ibc_middleware_test.go

```diff
@@ -24,6 +24,8 @@ import (
 	ibcmock "github.com/cosmos/ibc-go/v8/testing/mock"
 )
 
+var emptyForwardingPacketData = transfertypes.ForwardingPacketData{}
```

Comment:
I like that idea

Reply:
Same for `Forwarding` going into the `MsgTransfer`, looks like a lot of `transfertypes.Forwarding{}` diffs here! There's a good bit of that littered around now.

I would also suggest that I'd be fine with having a `WithForwarding(forwarding Forwarding)` method on the msg or packet data for assigning the forwarding, since it is an optional feature. Would be curious to hear what others think. This is obviously something that can be dealt with post-merge of this PR too. Would be happy stab the refactor myself 🗡️

Reply:
think I would prefer to keep the message signature the same and have it as a required field during construction, but have an obvious "I don't care about this" value.